src/tests/ftest/recovery/pool_list_consolidation.py

```diff
@@ -351,38 +359,48 @@ def test_lost_all_rdb(self):
         :avocado: tags=recovery,cat_recov,pool_list_consolidation
         :avocado: tags=PoolListConsolidationTest,test_lost_all_rdb
         """
-        # 1. Create a pool.
+        self.log_step("Create a pool.")
         self.pool = self.get_pool()
 
-        # 2. Stop servers.
+        self.log_step("Stop servers.")
         dmg_command = self.get_dmg_command()
         dmg_command.system_stop()
 
-        # 3. Remove /mnt/daos/<pool_uuid>/rdb-pool from all ranks.
-        hosts = list(set(self.server_managers[0].ranks.values()))
-        nodeset_hosts = NodeSet.fromlist(hosts)
-        command = f"sudo rm /mnt/daos0/{self.pool.uuid.lower()}/rdb-pool"
-        remove_result = pcmd(hosts=nodeset_hosts, command=command)
+        self.log_step("Remove <scm_mount>/<pool_uuid>/rdb-pool from all ranks.")
+        scm_mount = self.server_managers[0].get_config_value("scm_mount")
+        rdb_pool_path = f"{scm_mount}/{self.pool.uuid.lower()}/rdb-pool"
+        rdb_pool_out = check_file_exists(
+            hosts=self.hostlist_servers, filename=rdb_pool_path, sudo=True)
+        if not rdb_pool_out[0]:
+            msg = ("MD-on-SSD cluster. Contents under mount point are removed by control plane "
+                   "after system stop.")
+            self.log.info(msg)
+            dmg_command.system_start()
+            # return results in PASS.
+            return
+        command = f"sudo rm {rdb_pool_path}"
+        remove_result = pcmd(hosts=self.hostlist_servers, command=command)
```

Comment:
pre-existing so not a blocker but FYI we should be replacing `pcmd` with `run_remote` when we can